build.sh

```diff
@@ -3,28 +3,101 @@
 # Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
 #-------------------------------------------------------------------------------------------------------
 
-if [ ! -d "BuildLinux" ]; then
-    mkdir BuildLinux;
-fi
+SAFE_RUN() {
+    SF_RETURN_VALUE=$($1 2>&1)
```

Comment:
Can you use `local` and return this string instead of setting it to a global variable in this function?